### DISCUSSION TITLE
Abogabot en progreso

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#153] [Show and tell]
Hola les quiero compartir como va mi proyecto, todo lo he estado haciendo en figma por que es el que mas le entiendo por el momento comenten que opinan y en que podría mejorar, saludos :D
https://github.com/DaveOval/LauncheX_Abogabot/blob/main/Abogabot.md

**Comment by doguedogue** [comment]
Se ve genial 🚀🚀🚀

**Comment by VictorFalconi** [comment]
Esta interesante, bro. Buen avance

**Comment by Neo651** [comment]
Buena, recien voy a iniciar y me da una idea de como atacarlo. Gracias por compartir se ve buenisimo!!

**Comment by EsthebanDiego** [comment]
Se ve bien 👍

**Comment by Quiquebautista** [comment]
voy teniendo acceso y empezando mis practicas y woow la creatividad de todos aqui esta en otro nivel 🚀

**Comment by Alandcejas01** [comment]
Se ve muy bien, ¿con que pagina hiciste el diagrama de flujo? yo no entendía mucho de la practica pero gracias a la gente que va subiendo sus avances voy aprendiendo muchas gracias por eso.

**Reply by DaveOval** [reply]
Lo hice todo en figma, es la que más le se para diseño de cosas así aunque creo que no es la herramienta indicada para mi funciono muy bien

**Comment by XLoxxoX** [comment]
Disculpa una pregunta la platilla de la macbook de donde la sacaste? Me gustaría igual poder utilizarla para mi diseño, te lo agradecería mucho

**Reply by DaveOval** [reply]
Le pones en google "macbook mockup figma" y ahí salen muchas variedades igual en el figma qué compartir creo que también puedes copiar la plantilla

**Reply by DaveOval** [reply]
https://www.figma.com/community/file/839420676062339332 ahí le das en get a copy
